wazo_call_logd/utils.py

```diff
@@ -12,3 +14,33 @@ def find(col: Iterable[T], pred: Callable[[T], bool]) -> T | None:
     for x in col:
         if pred(x):
             return x
+
+
+class defaultdict(OrderedDict):
```

Comment:
Suggestion: `OrderedDefaultDict` to avoid confusion with the builtin `defaultdict`